daemon/emer-network-send-provider.c

```diff
@@ -191,15 +191,19 @@ reset_network_send_data (EmerNetworkSendProvider *self)
                   error->message);
       g_error_free (error);
     }
+
+  priv->send_number = 0;
+  priv->data_cached = TRUE;
 }
 
-static gboolean
+static void
 read_network_send_data (EmerNetworkSendProvider *self)
 {
   EmerNetworkSendProviderPrivate *priv =
     emer_network_send_provider_get_instance_private (self);
+
   if (priv->data_cached)
```

Comment:
Just ran make check 100 times in a for loop, and indeed `priv->data_cached` is always initialized to `0`. For the record though, uninitialized memory is highly likely to be 0 (largely because the kernel initializes new pages to 0 to prevent one process from reading another's memory). 0 is so much more likely than other values that a clever academic demonstrated that performance could be improved by guessing that any given attempt to read RAM would return 0 and backpedaling on the off chance that it was something else.